clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -2057,6 +2058,16 @@ class CIRCmpOpLowering : public mlir::OpConversionPattern<mlir::cir::CmpOp> {
   }
 };
 
+static mlir::LLVM::CallIntrinsicOp
+createCallLLVMIntrinsicOp(mlir::ConversionPatternRewriter &rewriter,
```

Comment:
There are other uses of `mlir::LLVM::CallIntrinsicOp` in this file, can you please replace them to use this helper in a follow up PR?

Reply:
I'd usually ask people to change as part of the PR, but I know you are tackling it later :)